package.json

```diff
@@ -45,16 +45,15 @@
     "jsonwebtoken": "^9.0.0",
     "jwk-to-pem": "^2.0.5",
     "mssql": "^10.0.1",
-    "notifications-node-client": "^7.0.3",
+    "notifications-node-client": "^7.0.6",
```

Comment:
if upgrading the package does not resolve the Snyk issue ,  should we keep the same version as there is no benefit on upgrade the package

Reply:
This resolves the axios issue and removes the need for the axios override that I also removed